### ISSUE TITLE
OracleSQL: Error using "Execute SQL Statement" 

### ISSUE THREAD

**Issue by awittenb** [dbeaver/dbeaver#4786]
Issue: I just upgraded from 5.1.x to 5.3.0 and when trying to run "Execute SQL Statement" (Ctrl+Enter), I'm receiving the following error:

```
Query execution failed

Reason:
SQL Error [6550] [65000]: ORA-06550: line 11, column 3:
PLS-00103: Encountered the symbol "end-of-file" when expecting one of the following:

   ; <an identifier> <a double-quoted delimited-identifier>
The symbol ";" was substituted for "end-of-file" to continue.
```

Example query:
```
DECLARE
  PI_ORDER NUMBER;
  PO_MESSAGE VARCHAR2(200);
BEGIN
  PI_ORDER := NULL;
  PO_MESSAGE := NULL;
  USER.PROCEDURE ( PI_ORDER , PO_MESSAGE );
  DBMS_OUTPUT.PUT_LINE( PO_MESSAGE );
  COMMIT;
END;
/
```

Action: Place cursor somewhere inside the statement and press Ctrl+Enter.

Observations: When selecting the statement first (without the /), I am able to execute the statement. It seems like there might be a parsing issue when not selecting the statement first.

I tried also downgrading to 5.2.5 and had the same error, so this appears to have started in the 5.2.x release. Downgrading back to 5.1.6 fixed the issue.

**Comment by SomeSander** [comment]
I wanted to confirm that I am experiencing the same issue in 5.3.0. I reverted back to 5.2.x which didn't work, hence tried to debug my script which wasn't touched recently. Luckily I found this issue and I would like to thank awittenb, as selecting the script and then running it works fine (control+enter or the run icon).

**Comment by Efus10n** [comment]
Same issue here with 5.3.0 on Oracle 11g (11.2.0.3.0). I keep an install of 5.2.0 on hand which still works fine.

**Comment by Efus10n** [comment]
should be fixed in 5.3.2 from duplicate #4868

**Comment by serge-rider** [comment]
@Efus10n thanks for reminding, I've missed this one.
BTW you can test it in Early Access version (https://dbeaver.io/files/ea)
